nomenclature/definitions/variable/technology/electricity-grid.yaml

```
@@ -1,25 +1,90 @@
 # List of variables related to the electricity grid
+# description of variables related to electricity demand
+Value of Lost Load|Electricity:
+   description: Cost of demand-not-served. Cost of load curtailment. Value of Lost Load (VoLL)
+   unit: €/MWh
 
-Network|Electricity|Demand|Reserve|Frequency Containment:
-   description: demand in frequency containment reserve (FCR)            
+Demand not served|Electricity:
+   description: Demand-not-served.
+   unit: MWh
+
+Reserve|Electricity|Requirement|Frequency Containment:
+   description: Demand in frequency containment reserve (FCR)
    unit: MW
-   
-Network|Electricity|Demand|Reserve|Automatic Frequency Restoration:
-   description: demand in automatic frequency restoration reserve (AFRR)             
+
+Reserve|Electricity|Requirement|Automatic Frequency Restoration:
+   description: Demand in automatic frequency restoration reserve (AFRR)
    unit: MW
-   
+
 Network|Electricity|Demand|Inertia:
```

```suggestion
Network|Electricity|Requirement|Inertia:
```
to make it consistant with reserves